w3-plan.md

```diff
@@ -0,0 +1,121 @@
+# Space
+
+![status:reliable](https://img.shields.io/badge/status-reliable-green.svg?style=flat-square)
+
+## Editors
+
+- [Vasco Santos], [Protocol Labs]
+
+## Authors
+
+- [Vasco Santos], [Protocol Labs]
+
+## Abstract
+
+A Plan that accounts can register to, in order to have an associated billing plan for their spaces.
```

Comment:
Hmm, ok how about:

```suggestion
A plan specifies limits and restrictions on usage. Different plans have different limits and different price points. Accounts select a plan so that they can be billed the correct amount each period.
```